sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/CHANGELOG.md

```diff
@@ -1,15 +1,22 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0 (2023-10-26)
 
 ### Features Added
 
+- Private Link support for DesktopVirtualization, more information can be found [here](https://learn.microsoft.com/en-us/azure/private-link/private-link-overview).
+- Scaling Plan Personal Schedules Added.
+
 ### Breaking Changes
 
+- Scaling Parameter "rampDownCapacityThresholdPct" minimum value changed from 0 to 1.
+
 ### Bugs Fixed
```

Comment:
I believe if we remove this subsection, the CI could pass. It is complaining that changelog should not have empty sections.